### PR TITLE
Matt arrays

### PR DESCRIPTION
**Pull request by matthewogtong**
Resolve all test cases for array functions in Array folder.

For `create2DArray`, I added an Array extension function at the top of the file right under the class. 

Code isn't the most elegant and could definitely be optimized but at least it passes the tests :/

### CODE REVIEW

File: 02Arrays/Arrays/Array/Array/Tester.swift [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+//  Created by Luat on 5/19/21.
+//
+
+import Foundation
+
+// Extension function used for create2DArray
+extension Array {
+    func chunked(by chunkSize: Int) -> [[Element]] {
+        return stride(from: 0, to: self.count, by: chunkSize).map {
+            Array(self[$0..<Swift.min($0 + chunkSize, self.count)])
```
Comment (LukeRave) [inline 02Arrays/Arrays/Array/Array/Tester.swift:14]: I don't think you need Swift.min between (0, 4, 8, 12 and self.count)

Comment (LukeRave) [reply 02Arrays/Arrays/Array/Array/Tester.swift:14]: nice work

**Comment by LukeRave** [comment]
nice work

File: 02Arrays/Arrays/Array/Array/Tester.swift (hunk 1 above)
Comment (matthewogtong) [reply 02Arrays/Arrays/Array/Array/Tester.swift:14]: Gotcha, thank you for clarifying. 
I'll adjust this extension function accordingly for future uses in my notes.

**Pull request closed without merging**